test/e2e/kappcontroller/packageinstall_test.go

```diff
@@ -624,13 +624,19 @@ spec:
 		}
 
 		tmpl := cr.Spec.Template[0]
-		if tmpl.Ytt != nil && len(tmpl.Ytt.Paths) != 0 {
-			if tmpl.Ytt.Paths[0] != "file.yml" {
-				t.Fatalf("\nExpected App spec.template.ytt.paths to contain file.yml\nGot: %s", tmpl.Ytt.Paths[0])
+
+		if tmpl.Ytt != nil {
```

Comment:
I am not sure how this change is related to the rest of the PR 🤔

Reply:
There were test case failing while running locally. Hence, added this change.